utils/prof_utils.py

```diff
@@ -0,0 +1,1150 @@
+#!/usr/bin/env python3
+
+import numpy as np
+import re
+import sys
+import logging
+import argparse
+from scipy.interpolate import UnivariateSpline
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+from astropy.time import Time
+from scipy.optimize import curve_fit
+
+from stickel import Stickel
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused Stickel imported from stickel](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4645369)